tasks/ca/statcan/geo.py

```diff
@@ -99,20 +89,25 @@
     GEO_FSA: 'cfsauid',
 }
 
+YEAR_URL = {
+    # http://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/bound-limit-2011-eng.cfm
+    '2011': 'http://www12.statcan.gc.ca/census-recensement/{year}/geo/bound-limit/files-fichiers/g{resolution}000b11{format}_e.zip',
+    # https://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/bound-limit-2016-eng.cfm
+    '2016': 'http://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/files-fichiers/{year}/l{resolution}000b16a_e.zip',
+}
+
 
-# http://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/bound-limit-2011-eng.cfm
 # 2011 Boundary Files
 class DownloadGeography(RepoFileUnzipTask):
 
     resolution = Parameter(default=GEO_PR)
-    year = Parameter(default="2011")
-
-    URL = 'http://www12.statcan.gc.ca/census-recensement/{year}/geo/bound-limit/files-fichiers/g{resolution}000b11{format}_e.zip'
+    year = Parameter()
 
     def get_url(self):
-        return self.URL.format(year=self.year,
-                               resolution=self.resolution,
-                               format=GEOGRAPHY_FORMAT[self.resolution])
+        url = YEAR_URL[self.year]
+        return url.format(year=self.year,
```

Comment:
Maybe we can remove the `year` placeholder since the URL is different per-year anyway.